vnext/template/cs-app/proj/MyApp.csproj

```diff
@@ -95,6 +95,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
```

Comment:
I added this to template bc it seems like a better default, esp with recent discussion of pinning to avoid supply-chain attacks.